web/core/local-db/worker/wa-sqlite/src/OPFSCoopSyncVFS.js

```
@@ -0,0 +1,592 @@
+  /** @type {Set<string>} */ accessiblePaths = new Set();
+  releaser = null;
+
+  static async create(name, module) {
+    const vfs = new OPFSCoopSyncVFS(name, module);
+    await Promise.all([vfs.isReady(), vfs.#initialize(DEFAULT_TEMPORARY_FILES)]);
+    return vfs;
+  }
+
+  constructor(name, module) {
+    super(name, module);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)